.gitignore

```diff
@@ -3,8 +3,4 @@
 /log/*.log
 /pkg/
 /tmp/
-/test/dummy/db/*.sqlite3
-/test/dummy/db/*.sqlite3-*
-/test/dummy/log/*.log
-/test/dummy/storage/
-/test/dummy/tmp/
+/spec/internal/
```

Comment:
maybe add a comment about this being for combustion?

Also do we want to include Gemfile.lock here?

Reply:
Ideally, Gemfile.lock is among the tracked files so the bundle environment will be similar among devs for situations like running the specs and building the gem.

Added comment ✅